tests/mfx_dispatch/linux/CMakeLists.txt

```diff
@@ -30,19 +30,25 @@ endif( )
 add_definitions( -DMFX_MODULES_DIR="${MFX_MODULES_DIR}" )
 add_definitions( -DMFX_PLUGINS_CONF_DIR="${MFX_PLUGINS_CONF_DIR}" )
 
+set( CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BIN_DIR}/${CMAKE_BUILD_TYPE})
```

Comment:
Try to avoid redefinition CMAKE_RUNTIME_OUTPUT_DIRECTORY or at least check whether it is defined by the user. This is cmake standard variable  https://cmake.org/cmake/help/v3.0/variable/CMAKE_RUNTIME_OUTPUT_DIRECTORY.html?highlight=cmake_runtime_output_directory